gui/slick/js/ajax-episode-subtitles.js

```diff
@@ -63,3 +63,62 @@
         });
     };
 })();
+
+(function() {
+    $.ajaxEpRedownloadSubtitle = function() {
+        $('.epRedownloadSubtitle').on('click', function() {
+            event.preventDefault();
+ 
+            // Check if we have disabled the click
+            if ($(this).prop('enableClick') === '0') { return false; }
+ 
+            selectedEpisode = $(this);
+ 
+            $("#confirmSubtitleReDownloadModal").modal('show');
+ 
+        });
+ 
+        $('#confirmSubtitleReDownloadModal .btn.btn-success').on('click', function(){
+            redownloadSubtitles();
+        });
+
+        function redownloadSubtitles() {
+            var loadingImage = 'loading16.gif';
+            var noImage = 'no16.png';
+            var yesImage = 'yes16.png';
+
+            var parent = selectedEpisode.parent();
+
+            // Create var for anchor
+            var link = selectedEpisode;
+
+            // Create var for img under anchor and set options for the loading gif
+            var img = selectedEpisode.children('img');
+            img.prop('title', 'loading');
+            img.prop('alt', '');
+            img.prop('src', 'images/' + loadingImage);
+
+            var url = selectedEpisode.prop('href');
+
+            $.getJSON(url, function(data) {
+                if (data.result.toLowerCase() === 'failure') {
+                    img.prop('src', 'images/' + noImage);
+                } else {
+                    img.prop('src', 'images/' + yesImage);
+                    disableLink(link);
+                }
+
+                // put the corresponding image as the result of queuing of the manual search
+                img.prop('title', imageResult);
+                img.prop('alt', imageResult);
+                img.prop('height', options.size);
+                img.prop('src', 'images/' + imageName);
```

Comment:
That looks right, if it works then that's good. I'll fix it all up at a later date.